include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -3325,10 +3325,11 @@ FEEvaluationBase<dim, n_components_, Number, is_face, VectorizedArrayType>::
       for (unsigned int v = 0; v < n_lanes; ++v)
         if (cells[v] != numbers::invalid_unsigned_int &&
             this->dof_info->hanging_node_constraint_masks.size() > 0 &&
-            this->dof_info->hanging_node_constraint_masks
-                [cells[v] * this->n_fe_components +
-                 this->first_selected_component] !=
-              internal::MatrixFreeFunctions::ConstraintKinds::unconstrained)
+            this->dof_info
+                ->hanging_node_constraint_masks[this->cell * n_lanes + v] !=
```

Comment:
Is there a reason you go away from `cells[v]`?

Reply:
No. There have been just too many merge conflicts... Let me fix it!

Reply:
Done!